src/reducers/index.js

```diff
@@ -217,3 +219,15 @@ export function getTasksErrorMessage(state) {
 export function isFetchingTasks(state) {
   return taskSelectors.isFetchingTasks(state.tasks);
 }
+
+export function getSecrets(state, namespace = getSelectedNamespace(state)) {
```

Comment:
second param should be an object, matching pattern used in other selectors and making this easier to extend / modify in future